Makefile

```diff
@@ -135,3 +135,7 @@ endif
 ## Experimental ##
 spec-ci:
 	docker-compose -f docker-compose.ci.yml run integration_ci bundle exec parallel_rspec spec --exclude-pattern 'spec/features/save_and_return_module_spec.rb' -n 4
+
+## Clears out emails older than 24 hours from the gmail inbox that receives test submissions
+clear-emails:
+	ruby ./integration/spec/support/email_output_service/clear_emails.rb
```

Comment:
On the [spec helper](https://github.com/ministryofjustice/fb-acceptance-tests/blob/master/integration/spec/spec_helper.rb#L36) all files is support will be required. Does this mean that when the specs are loading, requiring this script will delete the emails every time? *I am fine with that by the way, just double-checking if it is intent.

Reply:
Nope that was not my intent! 😆 

I'll move it into a different directory. Though come to think of it, instead of a scheduled job we could have it run as the last thing that every acceptance tests run does?

Reply:
Sounds a great idea! 👍